### DISCUSSION TITLE
Importing .osc EBSD data

### DISCUSSION THREAD

**Discussion by ute23** [mtex-toolbox/mtex#1937] [Ask Anything]
Got the following error when importing the .osc file
```
Warning: more column data was passed in than expected. Check your column names make sense!
Error using  & 
Matrix dimensions must agree.

Error in EBSD/subsind (line 84)
    ind = ind(:) & subs{i}(:);

Error in EBSD/subsref (line 34)
    ind = subsind(ebsd,s(1).subs);

Error in WizardProgressApi/localPlotDataCallback (line
111)
      ori = data(ind).orientations;
 
Error while evaluating UIControl Callback.
```

**Comment by aminsakic** [comment]
I have the same problem! Although my colleagues can load the same file. Is it a problem with the recent version of Mtex (MTEX 6.0.beta.1 9/2023)?

**Reply by kilir** [reply]
Could you share a file which triggers the error?

Cheers,
Rüdiger

**Comment by ute23** [comment]
Yes,

Please see the attached file.

 AM 3_19_Post DIC_1200X_13Dec22.oim
<https://drive.google.com/file/d/14DUziUimHGqR1lcJOi-yRLGQPVmcK0lS/view?usp=drive_web>

Roger Beal

**Reply by kilir** [reply]
Hi,
it seems, you may have to grant access to the file.
Cheers,
Rüdiger
